integration/experiment/power_sweep/power_sweep.py

```diff
@@ -100,7 +100,11 @@ def launch_configs(output_dir, app_conf, agent_types, min_power, max_power, step
     for power_cap in range(max_power, min_power-1, -step_power):
         for agent in agent_types:
             name = '{}'.format(power_cap)
-            options = {'power_budget': power_cap}
+            options = {'POWER_PACKAGE_LIMIT_TOTAL': power_cap}
+            if agent.startswith('power_balancer-'):
```

Comment:
The user can specify it on the command line.

Reply:
I think this should be one of the argparse options for invoking this py file like the other policy params are derived.  Mashing the period into the agent name seems hacky.

Reply:
Yeah that makes a lot of sense.  Especially because, as it is written, a user might try to specify "power_balancer,power_balancer-0.5,power_governor", but if they do that the output of the second balancer run overwrites the first one.  I know that by experience.